src/components/UniversalTable/UniversalTable.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../app/variables.scss";
+
+.TableContainer {
+  margin: 100px auto;
+  color: $tableMain;
+  width: $contentContainerWidth;
+
+  
+  .TableHeader:nth-child(odd) {
+    background-color: $lightGray;
+  }
+  .TableRow:nth-child(odd) {
+    background-color: $lightGray;
+  }
+  
+  .TableRow:nth-child(even) {
+    background-color: $mainGray;
+  }
+}
+
+.TableHeader {
+  position: relative;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+  z-index: 1;
+  box-shadow: 0px 3px 3px rgba(0, 0, 0, 0.458);
+  &Cell {
+    padding-left: 20px;
+    text-align: start;
+    
+  }
+}
+
+.TableRow {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+
+  &Cell {
+    padding-left: 20px;
```

Comment:
Styles for cell are repeated twice: here and in table header. Please make one common class for cell.

Reply:
Done